pkg/cmd/cli/cmd/rsync/rsync.go

```diff
@@ -229,7 +236,76 @@ func (o *RsyncOptions) Validate() error {
 
 // RunRsync copies files from source to destination
 func (o *RsyncOptions) RunRsync() error {
-	return o.Strategy.Copy(o.Source, o.Destination, o.Out, o.ErrOut)
+	err := o.Strategy.Copy(o.Source, o.Destination, o.Out, o.ErrOut)
+	if err != nil || !o.Watch {
```

Comment:
Wouldn't it be easier to read if the second condition stood on its own? It took me some effort to realize why `err != nil || !o.Watch`, and then the `err` in the next line might be nil or non-nil.

The more familiar idiom I see around would be:

```go
if err := o.Strategy.Copy(o.Source, o.Destination, o.Out, o.ErrOut); err != nil {
	return err
}

if !o.Watch {
	// Not going to watch for file changes, we're done.
	return nil
}

return o.WatchAndSync()

// ...
```

Just a suggestion.